src/components/admin/app-navbar/AppNavbar.vue

```diff
@@ -42,6 +42,11 @@
         required: true
       }
     },
+    methods: {
+      hideSidebar () {
```

Comment:
this is strange a little, so many repetitive @toggleMenu events. what do you think @asvae?